gracc-collector.spec

```diff
@@ -7,6 +7,7 @@ License:        MIT
 URL:            https://gracc.opensciencegrid.org
 Source:         %{name}.tar.gz
```

Comment:
Is there any existing way to extract a version number as part of the Makefile?

If not, I would set a test-environment-specific macro in the rpmbuild command in the makefile and `%ifdef` things.